src/cmd/dev.go

```diff
@@ -23,17 +23,44 @@ import (
 	"github.com/defenseunicorns/zarf/src/pkg/utils/helpers"
 	"github.com/mholt/archiver/v3"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
 var extractPath string
 
-var prepareCmd = &cobra.Command{
-	Use:     "prepare",
-	Aliases: []string{"prep"},
-	Short:   lang.CmdPrepareShort,
+var devCmd = &cobra.Command{
+	Use:     "dev",
+	Aliases: []string{"prepare"},
+	Short:   lang.CmdDevShort,
 }
 
-var prepareTransformGitLinks = &cobra.Command{
+var devDeployCmd = &cobra.Command{
+	Use:   "deploy",
+	Args:  cobra.ExactArgs(1),
```

Comment:
```suggestion
	Args:  cobra.MaximumNArgs(1),
```

This should be consistent with other commands that interact with package definitions.